src/packages/ant-form-helpers/FormScreen/index.js

```diff
@@ -1,54 +1,46 @@
-import React, { Component } from 'react';
+import React, { useEffect, useState } from 'react';
 import PropTypes from 'prop-types';
 import flattenObject from 'flat';
 
 export const FormContext = React.createContext();
 
-export default class FormScreen extends Component {
-  static propTypes = {
-    form: PropTypes.object.isRequired,
-    onSubmit: PropTypes.func.isRequired,
-  };
+const FormScreen = ({ form, children, onSubmit }) => {
+  const [isMounted, setIsMounted] = useState(false);
 
-  state = {
-    mounted: false,
-  };
+  useEffect(() => {
```

Comment:
are you sure that useEffect is enough? Maybe we want useLayoutEffect

Reply:
IDK how where `form. getFieldsError` reads from. Checking too soon for errors (rendered and not mounted) could mean that we render form without errors while it actually has some...

But I guess it's ok.